src/routes/product.svelte

```diff
@@ -2,119 +2,127 @@
 import Header from '$lib/header.svelte'
 import Footer from '$lib/footer.svelte'
 import Button from '$lib/button.svelte'
+import Section from '$lib/section.svelte'
 </script>
 <Header />
 <main class="md:ml-24 md:mr-44" >
   <section id="summary" class="md:flex md:relative" >
     <div class="md:w-1/3 md:h-96">
-      <h1 class="text-3xl pl-4 mt-8 mb-4 md:pl-0 md:mt-24 md:text-6xl ">hyper <br /><b>Products</b></h1>
-      <p class="text-base mx-4 text-darkgray md:mx-0">hyper provides a single entry point to five core application services, these services can be composed to effectively build any application.</p>
+      <h1 class="text-3xl pl-4 mt-8 mb-4 md:pl-0 md:mt-24 md:text-6xl ">hyper <br /><b>services</b></h1>
+      <p class="text-base mx-4 text-darkgray md:mx-0">hyper provides a single API entry point to core application services, these services can be composed to effectively build any application.</p>
```

Comment:
Build any application by taking advantage of hyper's single API entry point and composable application services.